### PR TITLE
Add docs regarding array updates by elements

### PR DESCRIPTION
**Pull request by jeeminso**
Follows: #14140

### CODE REVIEW

File: docs/sql/statements/update.rst [hunk 1]
```diff
@@ -51,7 +51,8 @@ Parameters
 :column_ident:
     The name of a column in the table identified by ``table_ident``. It is also
     possible to use :ref:`object subscript <sql-object-subscript>` to address
-    the inner fields of an object column.
+    the inner fields of an object column and
+    :ref:`array subscript <sql-array-subscript>` to address array by elements.
```
Comment (matriv) [inline docs/sql/statements/update.rst:55]:
```suggestion
    :ref:`array subscript <sql-array-subscript>` to address elements of an array.
```

**Review by matriv — APPROVED** [review]
LGTM, thx, left a suggestion for better phrasing, imho.

**Comment by matriv** [comment]
@mfussenegger Could you check this?

**Review by mfussenegger — APPROVED** [review]

**Comment by jeeminso** [comment]
retest this please

a flaky blackbox test:
```
> Task :blackbox:gtest
test_graceful_stop_concurrent_queries (test_decommission.TestGracefulStopDuringQueryExecution) ... Removed server http://127.0.0.1:4200 from active pool
```

**Comment by jeeminso** [comment]
another GH issue, retest this please

**Pull request merged**